src/packages/@ncigdc/utils/filters/index.js

```diff
@@ -235,17 +235,20 @@ export const makeFilter: TMakeFilter = fields => {
   if (!fields.length) return {};
   return {
     op: 'and',
-    content: fields.map(item => {
-      const value = _.isArray(item.value) ? item.value : item.value.split(',');
-
-      return {
-        op: 'in',
-        content: {
-          field: item.field,
-          value,
-        },
-      };
-    }),
+    content: fields
+      .filter(item => item && Object.keys(item).length !== 0)
```

Comment:
I'm not sure this is what should be changed (although having the extra check might not be a bad idea)? there is a filter being passed in that is just an empty object because there's no gene_id on the mutation page, and that's what's breaking here. I was thinking more that the geneFilter you're creating in the cancer distribution queries shouldn't be passed in unless it has values. What do you think?

Reply:
yes you are right. I will add something like this : if (c.content.field === 'genes.gene_id' && c.content.value)